src/update/orderbook_rewards.py

```diff
@@ -15,12 +15,13 @@ def dune_repr(df_row: Series) -> str:
     Specific to a COW Reward Item
     """
     return (
-        f"('{df_row['receiver'].lower()}', {df_row['num_trades']}, {df_row['amount']})"
+        f"('{df_row['receiver']}', {df_row['num_trades']}, {df_row['amount']})"
     )
 
 
 def rewards_df_to_dune_list(data: DataFrame) -> str:
     """Joins a list of VALUES into the format for postgres VALUES list"""
+    data["receiver"] = data["receiver"].apply(lambda x: x.lower())
```

Comment:
@bh2smith this was one of the proposals from #103 which I wanted to discuss one more time.

Reply:
Sounds good to me, although since we are iterating I don't really see the relevant difference. I kinda liked that the cast to lower was happening in the place that was actually responsible for doing the job of returning a dune representation.

Now the function simply trusts that the types are ok and can't guarantee anything.

Reply:
That's a valid point. Let's keep it as it was.